docs/athletes.md

```diff
@@ -1 +1,84 @@
-Athletes
\ No newline at end of file
+## **¿Qué es Shenanigan?** 🤸‍♀️
+
+
+Shenanigan es una plataforma de código abierto creada para atletas y por atletas. Estamos creando un mundo donde los atletas puedan recibir recompensas directas por superar sus propios límites. Shenanigan quiere brindarles a los atletas una plataforma para crecer, así como construir a su alrededor una comunidad que valore sus esfuerzos.
+
+
+## **¿Porque lo hace?** 🏆
+
+
+**Atleta** = del griego ‘’el que compite con esfuerzo por un premio’’
+
+
+Ahora que conoces la definición de atleta,  interiorízala, y pregúntate, ¿Por qué muchos atletas no tienen la oportunidad de conseguir ese premio ? ¿las oportunidades no son las mismas para todos? ¿solo algunos tienen el talento? ¿porque para algunos el esfuerzo no es proporcional a su premio? Shenanigan plantea ir más allá, la verdadera posibilidad de que cada atleta se ponga en marcha y compita por su premio. Shenanigan entrega a cada atleta los instrumentos web3 necesarios para lograr esto, ¡es tu momento! ¡es tu carrera!
+
+
+## **¿Cómo lo hace?** 📱
+
+
+Nuestros atletas descargan She_Dapp – interactúan con la interfaz y completan la información relevante para iniciar con un desafío.
+
+Ejemplo:
+
+•	**¿Que deporte practicas?**
+
+Atletismo
+
+•	**¿Qué categoría específica del deporte?**
+
+Media distancia - 1500m
+
+•	**Agrega un título para tu desafío**
+
+Correr 1500m en menos de 5 minutos
+
+•	**Describe el desafío que estás a punto de intentar**
+
+Quiero correr 1500m en menos de 5 minutos
+
+Después de completar esta información que será la data inicial de su desafío, Shenanigan acuña la NRG Card con su respectivo FAN Token, el FAN CLUB, y guarda el desafío para acuñar la CHALLENGE Card.
```

Comment:
I think this part might need a little more clarification. We could say "Since this is the first challenge under this category, we will mint their first "NRG Card" and add their first "Challenge Card" into the composable NRG container.

The NRG card is minted first, and the Challenge card is minted afterward once there is consensus on the outcome of the challenge. The NRG card will then track all challenges linked, and let Particles know if they are missing any parts of the set.

Reply:
This is highly impactful. It's hard to explain, but if we do it right, it will help a lot with the big picture.